<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="restTab">
       <attribute name="title">
        <string>Rest</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1">
         <widget class="QTextEdit" name="restConsole">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="restVerticalLayout">
          <item>
           <widget class="QPushButton" name="nextRestBtn">
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playPauseRestBtn">
            <property name="text">
             <string>PlayPause</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="enableRepeatRestBtn">
            <property name="text">
             <string>Enable Repeat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dislikeCurrentRestBtn">
            <property name="text">
             <string>Dislike Current</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playRestBtn">
            <property name="text">
             <string>Play</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="enableRepeatOneRestBtn">
            <property name="text">
             <string>Enable Repeat One</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="toggleLikeRestBtn">
            <property name="text">
             <string>Toggle Like</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pauseRestBtn">
            <property name="text">
             <string>Pause</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="likeCurrentRestBtn">
            <property name="text">
             <string>Like Current</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="previousRestBtn">
            <property name="text">
             <string>Previous</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="disableRepeatRestBtn">
            <property name="text">
             <string>Disable Repeat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="toggleShuffleRestBtn">
            <property name="text">
             <string>Toggle Shuffle</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="7">
         <layout class="QVBoxLayout" name="restGetVerticalLayout">
          <item>
           <widget class="QLabel" name="restGetLabel">
            <property name="text">
             <string>Get</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getInfoRestBtn">
            <property name="text">
             <string>Info</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getCurrentSongRestBtn">
            <property name="text">
             <string>Current Song</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getCurrentAlbumArtRestBtn">
            <property name="text">
             <string>Current Album Art</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getCurrentArtistRestBtn">
            <property name="text">
             <string>Current Artist</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getIsPlayingRestBtn">
            <property name="text">
             <string>Is Playing</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getRepeatStatusRestBtn">
            <property name="text">
             <string>Repeat Status</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getIsCurrentLikedRestBtn">
            <property name="text">
             <string>Is Currnet Liked</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="webSocketTab">
       <attribute name="title">
        <string>WebSocket</string>
       </attribute>
       <layout class="QGridLayout" name="webSocketGridLayout">
        <item row="0" column="0" rowspan="3">
         <layout class="QVBoxLayout" name="webSocketVerticalLayout">
          <item>
           <widget class="QPushButton" name="nextWebSocketBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playPauseWebSocketBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>PlayPause</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="enableRepeatWebSocketBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Enable Repeat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dislikeCurrentWebSocketBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Dislike Current</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playWebSocketBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Play</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="enableRepeatOneWebSocketBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Enable Repeat One</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="toggleLikeWebSocketBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Toggle Like</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pauseWebSocketBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Pause</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="likeCurrentWebSocketBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Like Current</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="previousWebSocketBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Previous</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="disableRepeatWebSocketBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Disable Repeat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="toggleShuffleWebSocketBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Toggle Shuffle</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1" rowspan="2" colspan="2">
         <widget class="QTextEdit" name="webSocketConsole">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="activateWebSocketBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Activate</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="3">
         <layout class="QVBoxLayout" name="webSocketGetVerticalLayout">
          <item>
           <widget class="QLabel" name="webSocketGetLabel">
            <property name="text">
             <string>Get</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getInfoWebSocketBtn">
            <property name="text">
             <string>Info</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getCurrentSongWebSocketBtn">
            <property name="text">
             <string>Current Song</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getCurrentAlbumArtWebSocketBtn">
            <property name="text">
             <string>Current Album Art</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getCurrentArtistWebSocketBtn">
            <property name="text">
             <string>Current Artist</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getIsPlayingWebSocketBtn">
            <property name="text">
             <string>Is Playing</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getRepeatStatusWebSocketBtn">
            <property name="text">
             <string>Repeat Status</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getIsCurrentLikedWebSocketBtn">
            <property name="text">
             <string>Is Currnet Liked</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="deactivateWebSocketBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Deactivate</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionActivateWebSocket"/>
    <addaction name="actionDeactivateWebSocket"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionActivateWebSocket">
   <property name="text">
    <string>Activate WebSocket</string>
   </property>
  </action>
  <action name="actionDeactivateWebSocket">
   <property name="text">
    <string>DeactivateWebSocket</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
